HISTORY.md

```diff
@@ -20,6 +22,7 @@
 * Add FileSystem::ReadAsync API in io_tracing
 * Add blob garbage collection parameters `blob_garbage_collection_policy` and `blob_garbage_collection_age_cutoff` to both force-enable and force-disable GC, as well as selectively override age cutoff when using CompactRange. 
 * Add an extra sanity check in `GetSortedWalFiles()` (also used by `GetLiveFilesStorageInfo()`, `BackupEngine`, and `Checkpoint`) to reduce risk of successfully created backup or checkpoint failing to open because of missing WAL file.
+* Add a new configuration option `blob_file_starting_level` in `AdvancedColumnFamilyOptions` and `MutableCFOptions` to enable writing blob files during flushes and compactions starting from the specified LSM tree level.
```

Comment:
Again, minor but `MutableCFOptions` is an internal implementation detail; we could simply say we added a new column family option (no need to refer to the actual class names)

Reply:
done